modules/integration/tests-ui-integration/src/test/java/org/wso2/carbon/mb/ui/test/dlc/DLCQueueTestCase.java

```diff
@@ -247,33 +289,63 @@ public boolean isElementPresent(String id) {
     }
 
     /**
-     * Search messageID through all messages in the queue
+     * Search messageID through all messages in a given queue.
+     *
+     * @param messageID searching messageID
+     * @param queueName searching queue
+     * @return true if the message is present
+     */
+    public boolean checkMessages(String messageID, String queueName) throws IOException {
+        //if the queue is the DLC, check messages in DLC content table
+        if (DLCQueueUtils.isDeadLetterQueue(queueName)) {
+            //browse the dlc
+            homePage.getDLCBrowsePage().getDLCContent();
+
+            //if the table isn't empty, check whether the message is present
+            if (isElementPresent(UIElementMapper.getInstance().getElement(DLC_MESSAGE_CONTENT_TABLE))) {
+
+                return checkMessagesInTable(driver.findElement(By.xpath(UIElementMapper.getInstance().
```

Comment:
Noted. Will be changed in all test cases throughout the test suite where applicable.